### PR TITLE
Made some tweaks to the documentation for the block upload functionality

### PR DESCRIPTION
**Pull request by 09jvilla**
In particular, called out the fact that we today don't validate blocks at upload time, though we do eventually intend to do this.

### CODE REVIEW

File: docs/sources/operators-guide/configure/configure-tsdb-block-upload.md
Comment (09jvilla) [inline docs/sources/operators-guide/configure/configure-tsdb-block-upload.md:10]: I wasn't familiar with `f.ex.` as an abbreviation. I figure its the same as `e.g.`? But haven't seen it used as much as e.g. so got rid of it.

**Review by pracucci — APPROVED** [review]
LGTM!

**Review by pstibrany — APPROVED** [review]
Thank you!

File: docs/sources/operators-guide/configure/configure-tsdb-block-upload.md
Comment (aknuds1) [reply docs/sources/operators-guide/configure/configure-tsdb-block-upload.md:10]: It's the same as e.g., yes.

**Review by aknuds1 — APPROVED** [review]
LGTM

**Comment by 09jvilla** [comment]
@treid314 once merged just tagging you for visibility so we can get this into the mimir 2.3 release.

**Comment by aknuds1** [comment]
Can we just merge this now?

**Comment by 09jvilla** [comment]
I think so? I don't have merge permissions :)

**Comment by aknuds1** [comment]
@09jvilla aha, I can take care of it then. We're used to the author merging, I guess that's why it hasn't been done.

**Pull request merged**
